content/W2015/W15-event-destress.md

```diff
@@ -1,4 +1,4 @@
-Title: Big CSters Discussion Circles: De-Stressing
+Title: Big CSters Discussion Circle: De-Stressing
 Date: 2015-03-05 18:00
```

Comment:
I liked the s because it gave the feeling of being part of a series
but that's just my opinion, idk if that's correct or whatever